.requirements

```diff
@@ -9,5 +9,5 @@ LUA_KONG_NGINX_MODULE=4d19e8d19c6dbc07eba5cf6f5ebacad95266f928 # 0.6.0
 LUA_RESTY_LMDB=222546680e9c31a9b97733b5db595688a521cd1a # 1.1.0
 LUA_RESTY_EVENTS=2f6fa23eb3d0b76a3b35fd915711200e90bc6732 # 0.1.6
 LUA_RESTY_WEBSOCKET=60eafc3d7153bceb16e6327074e0afc3d94b1316 # 0.4.0
-ATC_ROUTER=72cc8fddeac024c54c9c1fa5a25c28a72d79080e # 1.1.0
+ATC_ROUTER=main # 1.1.0
```

Comment:
Should be bumped to https://github.com/Kong/atc-router/releases/tag/v1.2.0.

Reply:
done